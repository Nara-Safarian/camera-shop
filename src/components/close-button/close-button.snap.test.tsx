import { render } from '@testing-library/react';
import CloseButton from './close-button';

test('CloseButton snapshot should be rendered correctly', () => {
  const {container} = render(
    <CloseButton onClose={() => {
      // nothing
    }}
    />
  );
  expect(container).toMatchSnapshot();
});
