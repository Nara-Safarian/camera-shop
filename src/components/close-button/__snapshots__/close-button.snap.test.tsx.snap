// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseButton snapshot should be rendered correctly 1`] = `
<div>
  <button
    aria-label="Закрыть попап"
    class="cross-btn"
    type="button"
  >
    <svg
      aria-hidden="true"
      height="10"
      width="10"
    >
      <use
        xlink:href="#icon-close"
      />
    </svg>
  </button>
</div>
`;
